### PR TITLE
Refactor to Python and Robocorp libs

### PR DESCRIPTION
**Pull request by cmin764**
This takes into use the new `robocorp-windows` library in order to automate Notepad by setting a text inside after configuring a custom font.
[Linear](https://linear.app/robocorp/issue/DAP-68/convert-example-windows-notepad-to-python)

Tested on both Windows 11 and Server 2019:

test-run-windows-notepad

https://github.com/robocorp/example-windows-notepad/assets/709053/8138a31a-bd1b-4c60-8cde-c588aa8b4010

### CODE REVIEW

**Comment by cmin764** [comment]
This is partially blocked (by having an uglier solution) due to this bug: https://github.com/robocorp/robocorp/issues/172

**Comment by cmin764** [comment]
Up next would be to update the README as well (once we're fine with the changes).

File: tasks.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+def _set_property(unique_id: str, value: str):
+    field = window.find(f'id:"{unique_id}"')
+    field.click()
+    field.send_keys("{DOWN}", wait_time=0.5)
+    desktop.send_keys(value, send_enter=True, wait_time=0.5)
+
+
+def _select_property(unique_id: str, value: str):
+    field = window.find(f'id:"{unique_id}"')
+    field.ui_automation_control.Select(value)
```
Comment (fabioz) [inline tasks.py]: In general the `ui_automation_control` field should not be accessed.

This method could be removed as the following could be written directly:  

`window.find(f'id:"{unique_id}"').select(value)`

or alternatively:

`window.select(value, locator=f'id:"{unique_id}"')`

File: tasks.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+OPEN_FILE = str(os.getenv("FILE_TO_OPEN", Path("devdata") / "test.txt").absolute())
+
+
+def get_win_version() -> str:
+    """Windows only utility which returns the current Windows major version."""
+    # Windows terminal `ver` command is bugged, until that's fixed, check by build
+    #  number. (the same applies for `platform.version()`)
+    version_parts = platform.version().split(".")
+    major = version_parts[0]
+    if major == "10" and int(version_parts[2]) >= 22000:
```
Comment (OvidiuCode) [inline tasks.py]: I think it would be better if we could use some constants instead of magic numbers

File: tasks.py (hunk 1 above)
Comment (kariharju) [reply tasks.py]: Agree with this, hide the "internals" as much as possible as these examples are the ones that ReMark will suggest and users will imitate.

**Review by kariharju — APPROVED** [review]
Fabio's comment I'd fix but other than that LGTM

File: tasks.py (hunk 1 above)
Comment (cmin764) [reply tasks.py]: And ReMark does a great job at it, maybe the first time I tried it it didn't lead me to the answer I'm getting now:

Screenshot 2024-01-23 at 1 42 41 PM

**Pull request merged**